crucible/src/Lang/Crucible/CFG/Reg.hs

```diff
@@ -676,12 +683,38 @@ data CFG ext s (init :: Ctx CrucibleType) (ret :: CrucibleType)
          , cfgNextLabel :: !Int
          -- ^ A number greater than any label ID appearing in the CFG.
          }
+   -- An instantiated CFG.
+   -- TODO: maybe we can fold this instantiation into the FnHandle type
+   | forall subst init' ret'.
+      (Instantiate subst init' ~ init, Instantiate subst ret' ~ ret) =>
+     ICFG { cfgIHandle :: !(FnHandle init' ret')
```

Comment:
I wonder... is there ever a good reason to have an instantiated version of a registerised CFG?

Reply:
You're right. This isn't necessary.